renovate.json5

```diff
@@ -0,0 +1,36 @@
+{
+  extends: [
+    "config:base",
+    ":disableRateLimiting",
+  ],
+  automerge: false,
+  platformAutomerge: false,
```

Comment:
IMO auto merge can be enabled. but it depends on your opinions.

https://docs.renovatebot.com/configuration-options/#platformautomerge

Reply:
Let's keep `automerge: false` for now until we feel more confident and can get the PR checks to always pass, then we can set it to `true` 🙏